superset-frontend/src/filters/components/Select/SelectFilterPlugin.tsx

```diff
@@ -16,6 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+/* eslint-disable no-param-reassign */
```

Comment:
thought: if we want to encourage use of `useImmerReducer`, we should probably disable this linting rule.

Reply:
not sure, if we need it in all places disabled... only where we use immer, so may be later